Automattic/wp-calypso
# Add LinkedIn Insights tag for Jetpack Cloud

#### Proposed Changes

* Clean up old unused implementation of LinkedIn
* Add LinkedIn Insights tag for Jetpack environment

#### Testing Instructions

* Checkout this PR
* Run `yarn start-jetpack-cloud`
* Open `http://jetpack.cloud.localhost:3000/pricing?flags=ad-tracking,cookie-banner`
* If in GDPR region, accept any cookie banner that will appear; reload the page afterward
* Check that `window.lintrk` (in console) is defined
* Run `yarn start` and go to `http://calypso.localhost:3000/plugins?flags=ad-tracking,cookie-banner`
* Check that `window.lintrk` (in console) is undefined

#### Pre-merge Checklist





Related to D95453-code
